### PR TITLE
ZENKO-3620 docker upgrade now requires double quote on volumes

### PR DESCRIPTION
**Pull request by tcarmet**

### CODE REVIEW

**Review by nicolas2bert — APPROVED** [review]

File: eve/workers/end2end.yaml [hunk 1]
```diff
@@ -4,6 +4,18 @@ kind: Pod
 metadata:
   name: "zenko-operator-worker-smoke"
 spec:
+  initContainers:
```
Comment (miniscruff) [inline eve/workers/end2end.yaml:8]: Do you think you could add some context to this, it looks pretty magical. Or maybe a follow up ticket to remove it when we can.

**Review by alexanderchan-scality — APPROVED** [review]

File: eve/workers/end2end.yaml (hunk 1 above)
Comment (tcarmet) [reply eve/workers/end2end.yaml:8]: Just added the link to the incident

**Review by miniscruff — APPROVED** [review]

**Pull request merged**